src/demo/debugging-demo.ts

```diff
@@ -248,68 +254,25 @@ async function main() {
           d3.zoomIdentity.translate(scene.offset.x, scene.offset.y)
               .scale(scene.scale.x));
 
-  let previouslyHoveredSprites = new Set<Sprite>();
-  function handleHitTestResult(result: HitTestResult) {
-    const newlyHoveredSprites = new Set<Sprite>(result.hits);
-
-    // Perform alterations on newly hovered sprites.
-    result.hits.forEach(sprite => {
-      if (sprite.isRemoved) {
-        // Short-circuit if this sprite is no longer valid.
-        return;
-      }
-      if (previouslyHoveredSprites.has(sprite)) {
-        // Short-circuit if this sprite was previously hovered.
-        return;
-      }
-      sprite.update(s => {
-        s.TransitionTimeMs = settings.transitionTimeMs;
-        s.BorderColor = hoverColor;
-      });
-    });
-
-    // Perform alterations on previously hovered sprites that are not still
-    // hovered.
-    [...previouslyHoveredSprites].forEach(sprite => {
-      if (sprite.isRemoved) {
-        // Short circuit if this sprite is no longer valid.
-        return;
-      }
-      if (newlyHoveredSprites.has(sprite)) {
-        // Short circuit if this sprite is newly hovered.
-        return;
-      }
-      sprite.update(s => {
-        s.TransitionTimeMs = settings.transitionTimeMs;
-        s.BorderColor = borderColor;
-      });
-    });
-
-    previouslyHoveredSprites = newlyHoveredSprites;
-  }
-
-  let hitTestTask: HitTestPromise|undefined = undefined;
-  container.addEventListener('mousemove', async (event) => {
-    if (hitTestTask) {
-      hitTestTask.cancel();
-      hitTestTask = undefined;
-    }
-
+  container.addEventListener('mousemove', (event) => {
     if (!settings.mouseover) {
```

Comment:
nit: setting name is `mouseover` but is attached to a `mousemove` event.

Reply:
Renaming the setting to make it more clear what it does.